Nos/Models/Event+CoreDataClass.swift

```diff
@@ -942,21 +942,16 @@ public class Event: NosManagedObject {
     }
 
     class func attributedContentAndURLs(
```

Comment:
These changes look fine to me, I do like that we get rid of the note as soon as possible by extracting te content and allTags, I don't remember why we were doing this long route. I suspect that it is because of fear to mix the managed object and the managed object context.

It could be useful if we document this function and add something like "the managed object context doesn't have to be the same context that manages the note, it is just used to create referenced authors and events while parsing".

Reply:
Good idea! I added the comment.